variables.tf

```diff
@@ -38,6 +38,12 @@ variable "autoscaler_role_name" {
   default     = "quortex-autoscaler"
 }
 
+variable "autoscaler_sa_name" {
+  type        = string
+  description = "Service Account name for Autoscaler"
+  default     = "cluster-autoscaler-sa"
+}
+
```

Comment:
An variablke with the follwing type would have been nice.
```
object({
  namespace = string
  name = string
})
```

Reply:
Really? The module is not managing Namespaces, nor k8s service accounts, I don't see the point?